marimbabot_vision/scripts/generate_data.py

```diff
@@ -203,14 +203,14 @@ def generate_piece(self,num_bars=3,):
             self.add_tempo(voice_1)
         if self.slurs:
             self.add_slurs(voice_1)
+        if self.include_dynamics:
```

Comment:
> Also fixes a placement error with dotted chords and a false "unregognized" message for repeats

Is this here?

Reply:
It was an attempt at changing the order of dynamics and articulations in the data generation (as this was causing issues in the handwritten data generation, because in a previous version, the order was switched). But it did not work unfortunately. I commited before I noticed the issue still occurs. I can switch it back if you'd like